web/src/left_sidebar_navigation_area.ts

```diff
@@ -129,7 +120,7 @@ export function handle_narrow_activated(filter: Filter): void {
         filter.operands("sender")[0] === people.my_current_email()
     ) {
         $filter_li = $(".top_left_my_reactions");
-        $filter_li.addClass("active-filter");
+        $filter_li.addClass("top-left-active-filter");
```

Comment:
This code path could probably call `select_top_left_corner_item` rather than calling it with the empty string at the top, right? 

I guess maybe some care might needed for an `else` clause to clear it if none of these match?